ci/nightly/pipeline.template.yml

```diff
@@ -95,10 +95,11 @@ steps:
                 - common-ancestor
       - id: scalability-benchmark-dml-dql
         label: "Scalability benchmark (read & write) against merge base or 'latest'"
-        depends_on: build-aarch64
+        depends_on: build-x86_64
         timeout_in_minutes: 240
         agents:
-          queue: linux-aarch64
+          # Larger instance is more stable in performance
+          queue: hetzner-x86-64-dedi-8cpu-32gb
```

Comment:
Did you mean to make this a `hetzner-x86-64-dedi-16cpu-64gb` like the others that have this comment?

Reply:
Not necessarily. I just saw that with a smaller instance it was running flakily.